### PR TITLE
srt_c_api: remove duplicate code in srt_epoll_*

### PR DESCRIPTION
**Pull request by dabrain34**
Remove specific define block for LINUX

### CODE REVIEW

File: srtcore/srt_c_api.cpp [hunk 1]
```diff
@@ -207,17 +207,11 @@ int srt_epoll_add_ssock(int eid, SYSSOCKET s, const int * events)
...
-	} else {
-        flag = SRT_EPOLL_IN | SRT_EPOLL_OUT | SRT_EPOLL_ERR;
-    }
-#elif defined(BSD) || defined(OSX) || (TARGET_OS_IOS == 1) || (TARGET_OS_TV == 1)
-    if (events) {
-        flag = *events;
-	} else {
-        flag = SRT_EPOLL_IN | SRT_EPOLL_OUT | SRT_EPOLL_ERR;
+    } else {
+	flag = SRT_EPOLL_IN | SRT_EPOLL_OUT | SRT_EPOLL_ERR;
```
Comment (maxlovic) [inline srtcore/srt_c_api.cpp]: tab -> space

File: srtcore/srt_c_api.cpp [hunk 2]
```diff
@@ -247,17 +241,11 @@ int srt_epoll_update_ssock(int eid, SYSSOCKET s, const int * events)
...
```
Comment (maxlovic) [inline srtcore/srt_c_api.cpp]: Replace a tab character with spaces.

**Comment by maxlovic** [comment]
`CEPoll::add_ssock` also contains conditional checks. Will need a more thorough review to probably clean up the rest code as well.

**Comment by ethouris** [comment]
We need this situation to be clarified in the next 10 days. With no changes up to that time this PR will be abandoned.

**Pull request merged**